packages/fxa-auth-server/scripts/populate-stripe-customer-db.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ts-node-script
```

Comment:
If this is using ts-node, shouldn't it be a TypeScript file?

Reply:
I was following the convention of the other scripts and was uncertain if it should be .ts or .js. I can update that.